test/Twilio.Test/Converters/SerializersTest.cs

```diff
@@ -41,5 +42,15 @@ public void TestJsonObjectPassesThroughString()
             var result = Serializers.JsonObject(input);
             Assert.AreEqual(input, result);
         }
+
+        [Test]
+        public void TestDateTimeIso8601()
+        {
+            var expect = "2017-06-19T12:13:14Z";
+            var input = new DateTime(2017, 06, 19, 12, 13, 14);
+            var result = Serializers.DateTimeIso8601(input);
+            Assert.AreEqual(expect, result);
+        }
```

Comment:
Can you add a test for `Serializers.DateTimeIso8601(null)`? Your function signature implies it accepts null values.